fw/tls.c

```diff
@@ -1160,6 +1180,11 @@ tfw_tls_start(void)
 {
 	tfw_tls_allow_any_sni = allow_any_sni_reconfig;
 
+	if (tls_get_ja5_storage_size()) {
+		if (!ja5t_init_filter(tls_get_ja5_storage_size()))
```

Comment:
Save `tls_get_ja5_storage_size` result to avoid second call of this function

Reply:
done